releasetools/x86_hdimage.sh

```diff
@@ -24,6 +20,10 @@ then
 	exit 1
 fi
 
+# set up disk creation environment
+. releasetools/image.defaults
+. releasetools/image.functions
+
```

Comment:
Did you check that moving those above the HDD default size doesn't break it. I mean, if image.defaults sets {HOME,USR,…}_SIZE, the commands above will never be executed.